src/main/AuthenticationResponse.ts

```diff
@@ -268,6 +267,12 @@ async function createSIOPResponsePayload(
     vp_token,
     verifiable_presentations,
   };
+  if (supportedDidMethods.length && resOpts.did) {
```

Comment:
Hmzz, this doesn't sound right.
So if we have a DID we are using a jwk/thumbprint. Should be the other way around

Reply:
yes, this should be other way around